Microsoft.Toolkit.Uwp.Services/Services/Twitter/Tweet.cs

```diff
@@ -44,23 +38,78 @@ public string Text
         }
 
         /// <summary>
-        /// Gets or sets the extended mode.
+        /// Gets or sets text of the tweet (280 characters).
+        /// </summary>
+        [JsonProperty("full_text")]
+        private string FullText { get; set; }
+
+        /// <summary>
+        /// Gets or sets display text range (indexes of tweet text without RT and leading user mentions)
         /// </summary>
-        [JsonProperty("extended_tweet")]
-        public TwitterExtended Extended { get; set; }
```

Comment:
Removing the property is a breaking change. Mark as deprecated to be removed at a future major version